django/db/__init__.py

```diff
@@ -59,3 +62,13 @@ def close_old_connections(**kwargs):
 
 signals.request_started.connect(close_old_connections)
 signals.request_finished.connect(close_old_connections)
+
+
+# Register an event to close all connections before forking a process.
```

Comment:
I think this will need to be adjusted as well, connections should be closed in the process fork but remain opened in the parent.

Reply:
The comment is just incorrect – the function is registered `after_in_child` like my original issue suggests.